github/resource_github_repository_deploy_key.go

```diff
@@ -54,12 +55,13 @@ func resourceGithubRepositoryDeployKeyCreate(d *schema.ResourceData, meta interf
 	key := d.Get("key").(string)
 	title := d.Get("title").(string)
 	readOnly := d.Get("read_only").(bool)
-
 	owner := meta.(*Organization).name
+
+	log.Printf("[DEBUG] Creating repository deploy key: %s (%s/%s)", title, owner, repoName)
 	resultKey, _, err := client.Repositories.CreateKey(context.TODO(), owner, repoName, &github.Key{
-		Key:      &key,
-		Title:    &title,
-		ReadOnly: &readOnly,
+		Key:      github.String(key),
```

Comment:
Curious what is the advantage to using the helper?

Reply:
it just looks cleaner to me as we're making the SDK contract more visible/explicit 🤷‍♂️

Reply:
fair enough!